packages/flutter/lib/src/widgets/app.dart

```diff
@@ -70,6 +71,11 @@ class WidgetsApp extends StatefulWidget {
   /// named route.
   final RouteFactory onGenerateRoute;
 
+  /// The name of the first route to show.
+  ///
+  /// Defaults to [Window.defaultRouteName].
```

Comment:
The `dart:ui` import is prefixed with `ui`, so the link may need to be `[ui.Window.defaultRouteName]` to actually work. Or perhaps it should point to the `ui.window` variable instead of the `ui.Window` class?